config/proto/targets.proto

```diff
@@ -0,0 +1,32 @@
+syntax = "proto2";
+
+package hermes.targets
```

Comment:
```suggestion
package hermes.targets;
```

Reply:
I altered this package in a newer commit to make it:
package hermes;